modules/mobianRtdProvider.js

```diff
@@ -24,6 +22,21 @@ export const mobianBrandSafetySubmodule = {
 function init() {
   return true;
 }
+
+function safeJSONParse(input) {
```

Comment:
can you import this? https://github.com/prebid/Prebid.js/blob/465096c4ab99964e9210e969c404382e13142c3b/src/utils.js#L1107 and if that is problematic, could you pr the util instead of your adpter with the extra logic

Reply:
Thanks! I'll just import this, I haven't got any reason to believe that in fact the API response *ever* came in pre-parsed. It'll be a bit cause I need to fix the tests.